test/common/fam_test_config.h.in

```diff
@@ -60,6 +60,12 @@ using namespace openfam;
 // Runtime configuration
 #cmakedefine TEST_NPE "${TEST_NPE}"
 
+#cmakedefine TEST_SKIP_STATUS ${TEST_SKIP_STATUS}
+
+#ifndef TEST_SKIP_STATUS
+#define TEST_SKIP_STATUS 0    
+#endif
```

Comment:
You can set TEST_SKIP_STATUS at only one place, i.e, in CMakeLists.txt file.

uncomment line#63.